libraries/core-react/azure-pipelines.yml

```diff
@@ -3,17 +3,52 @@
 # Add steps that build, run tests, deploy, and more:
 # https://aka.ms/yaml
 
-trigger:
-- master
+name: eds-react-$(Build.BuildId)
 
 pool:
   vmImage: 'ubuntu-latest'
 
+trigger:
+  batch: 'true'
+  branches:
+    include:
+      - develop
+  paths:
+    include:
+      - libraries/core-react
+
+pr:
+  branches:
+    include:
+      - develop
+  paths:
+    include:
+      - libraries/core-react
+
 steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '14.x'
```

Comment:
Suggest we also bump version in `Dockerfile` for Storybook and Storefront from 12 to 14 to match whats being tested to be sure

Reply:
I agree, but I don’t think that belongs in this pr since this is about running testing and linting in the React library, but I can make a separate issue for that.

Reply:
Fair enough 👍